packages/fxa-auth-server/bin/update_firestore_from_oauth.js

```diff
@@ -50,8 +51,20 @@ async function main() {
     if (doc.exists) {
       const data = doc.data();
       if (data && data.oauth_clients && data.oauth_clients[CLIENT_ID]) {
-        // Record is already in the database
-        continue;
+        if (REMOVE_CLIENT) {
+          // Remove this entry
+          await document.set(
+            {
```

Comment:
So, did you want to test for case, and leave it alone if it is not uppercase?

Reply:
We know its already case sensitive, so this test is already also case sensitive. Building more fault detection in feels less useful as once this script is run and we get the appropriate results in prod, its going to be removed. This is a one-off script.

Reply:
Or is the intent to run with the uppercase CLIENT_ID value and REMOVE_CLIENT=true. If it finds a match using the uppercase, then nuke it.

Reply:
Yes, that was the intent. Use the upper-case version with REMOVE_CLIENT.

Reply:
Okay. Thanks.